spec/models/spree_amazon/response_spec.rb

```diff
@@ -368,6 +367,48 @@
           end
         end
       end
+      
+      describe '#soft_decline?' do
+        context 'it is null' do
```

Comment:
add `when` to "it is null", so that it will become `when it is null`, and also specify what is that `it` as it's hard to understand